samples/SpectatorView.Example.Unity/.gitignore

```diff
@@ -72,4 +72,7 @@ UWP/
 
 # Avoid checking in build directories
 Assets/BuildTools.Unity
-Assets/BuildTools.Unity.meta
\ No newline at end of file
+Assets/BuildTools.Unity.meta
+
+# Avoid checking in asset caches
+Assets/Generated.StateSynchronization.AssetCaches/
```

Comment:
There are currently a few assets in this directory that aren't generated. We need to move those elsewhere.  I believe they are:

SpectatorViewSettings.prefab
StateSynchronizationSettings.prefab
DefaultStateSynchronizationPerformanceParameters.prefab